examples/storm-starter/src/jvm/org/apache/storm/starter/ManualDRPC.java

```diff
@@ -30,39 +30,37 @@
 import org.apache.storm.tuple.Tuple;
 import org.apache.storm.tuple.Values;
 
-
 public class ManualDRPC {
-  public static class ExclamationBolt extends BaseBasicBolt {
-
-    @Override
-    public void declareOutputFields(OutputFieldsDeclarer declarer) {
-      declarer.declare(new Fields("result", "return-info"));
-    }
+    public static class ExclamationBolt extends BaseBasicBolt {
 
-    @Override
-    public void execute(Tuple tuple, BasicOutputCollector collector) {
-      String arg = tuple.getString(0);
-      Object retInfo = tuple.getValue(1);
-      collector.emit(new Values(arg + "!!!", retInfo));
-    }
+        @Override
+        public void declareOutputFields(OutputFieldsDeclarer declarer) {
+            declarer.declare(new Fields("result", "return-info"));
+        }
 
-  }
+        @Override
+        public void execute(Tuple tuple, BasicOutputCollector collector) {
+            String arg = tuple.getString(0);
+            Object retInfo = tuple.getValue(1);
+            collector.emit(new Values(arg + "!!!", retInfo));
+        }
 
-  public static void main(String[] args) {
-    TopologyBuilder builder = new TopologyBuilder();
-    LocalDRPC drpc = new LocalDRPC();
+    }
 
-    DRPCSpout spout = new DRPCSpout("exclamation", drpc);
-    builder.setSpout("drpc", spout);
-    builder.setBolt("exclaim", new ExclamationBolt(), 3).shuffleGrouping("drpc");
-    builder.setBolt("return", new ReturnResults(), 3).shuffleGrouping("exclaim");
+    public static void main(String[] args) throws Exception{
+        TopologyBuilder builder = new TopologyBuilder();
+        LocalDRPC drpc = new LocalDRPC();
 
-    LocalCluster cluster = new LocalCluster();
-    Config conf = new Config();
-    cluster.submitTopology("exclaim", conf, builder.createTopology());
+        DRPCSpout spout = new DRPCSpout("exclamation", drpc);
+        builder.setSpout("drpc", spout);
+        builder.setBolt("exclaim", new ExclamationBolt(), 3).shuffleGrouping("drpc");
+        builder.setBolt("return", new ReturnResults(), 3).shuffleGrouping("exclaim");
 
-    System.out.println(drpc.execute("exclamation", "aaa"));
-    System.out.println(drpc.execute("exclamation", "bbb"));
+        LocalCluster cluster = new LocalCluster();
+        Config conf = new Config();
+        cluster.submitTopology("exclaim", conf, builder.createTopology());
 
-  }
+        System.out.println(drpc.execute("exclamation", "aaa"));
+        System.out.println(drpc.execute("exclamation", "bbb"));
+    }
 }
```

Comment:
Please don't change the spacing/formatting for files you didn't touch otherwise.